pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -157,14 +184,47 @@ func (hc *HelmCheck) setupInformers(sharedInformerFactory informers.SharedInform
 	)
 }
 
-func helmTags(release *release, storageDriver helmStorage) []string {
+func (hc *HelmCheck) addExistingReleases(apiClient *apiserver.APIClient) error {
+	selector := labels.Set{"owner": "helm"}.AsSelector()
+
+	initialHelmSecrets, err := apiClient.Cl.CoreV1().Secrets(v1.NamespaceAll).List(
+		context.Background(), metav1.ListOptions{LabelSelector: selector.String()},
+	)
+	if err != nil {
+		return err
+	}
+
+	for _, secret := range initialHelmSecrets.Items {
+		hc.addSecret(&secret)
+	}
+
+	initialHelmConfigMaps, err := apiClient.Cl.CoreV1().ConfigMaps(v1.NamespaceAll).List(
+		context.Background(), metav1.ListOptions{LabelSelector: selector.String()},
+	)
+	if err != nil {
+		return err
+	}
+
+	for _, configMap := range initialHelmConfigMaps.Items {
+		hc.addConfigmap(&configMap)
+	}
+
+	hc.existingReleasesStored = true
```

Comment:
It this to prevent the first run from generating events? Would be nice to add a comment.

Reply:
Exactly. I added the comment in the struct definition: https://github.com/DataDog/datadog-agent/pull/11317/files#diff-a91a2bb9f4306899a82675022adbdb60dcdec035844e948284139007fc4dcb83R65